test/module/irohad/ametsuchi/storage_test.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include "ametsuchi/impl/storage_impl.hpp"
+
+using namespace iroha::ametsuchi;
+using namespace iroha::expected;
+
+class StorageTest : public ::testing::Test {
+ protected:
+  std::string block_store_path =
+      (boost::filesystem::temp_directory_path() / "block_store").string();
+  std::string dbname_ = "irohadb";
```

Comment:
I guess this name better to make random. For example pick UUID. It can be generated by boost.